go/client/cmd_chat_send.go

```diff
@@ -66,7 +68,15 @@ func newCmdChatSend(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comm
 }
 
 func (c *CmdChatSend) Run() (err error) {
-	return chatSend(context.TODO(), c.g, ChatSendArg{
+	err = annotateResolvingRequest(c.G(), &c.resolvingRequest)
```

Comment:
chose to mutate resolvingRequest instead of returning a new one, seems to be idiomatic in Go?